docs/release-notes/july-2024.md

```diff
@@ -0,0 +1,157 @@
+---
+slug: july-2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: July 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for July 2024
+
+## 🌐 Semgrep AppSec Platform
+
+### Added
+
+<!-- Azure DevOps, Bitbucket Cloud, Bitbucket Data Center -->
+
+### Changed
+
+### Fixed
+
+## 💻 Semgrep Code
+
+### Added
+
+- Added the `--exclude-minified-files` flag to enable skipping minified files and the `--no-exclude-minified-files` flag to include minified files during scans. By default, Semgrep scans minified files.
```

Comment:
Is this for `semgrep ci`, `semgrep scan`, or both?

Reply:
Both!